compiler/rustc_resolve/src/build_reduced_graph.rs

```diff
@@ -1462,6 +1473,14 @@ impl<'a, 'b> Visitor<'b> for BuildReducedGraphVisitor<'a, 'b> {
             self.visit_invoc(sf.id);
         } else {
             let vis = self.resolve_visibility(&sf.vis);
+            debug!(
+                "BuildReducedGraphVisitor::visit_struct_field: self.parent_scope={:?}",
+                self.parent_scope
+            );
+            debug!(
+                "BuildReducedGraphVisitor::visit_struct_field: sf={:?}, sf.vis={:?}, vis={:?}",
+                &sf, &sf.vis, &vis
+            );
```

Comment:
Could you remove this temporary debugging code?

Reply:
I had more debugging code that I did remove; i thought this might be helpful for debugging in the future.

Reply:
>I thought this might be helpful for debugging in the future.

It really won't be. (And in the unlikely case if it will, it could be added and removed again.)

Reply:
Okay, I'll remove it then.

Reply:
(I'm going to remove it after my build finishes.)